coops-ui/tests/unit/AddressListSm.spec.ts

```diff
@@ -4,79 +4,132 @@ import Vuelidate from 'vuelidate'
+      'deliveryAddress':
+        {
+          'streetAddress': '220 Buchanan St',
+          'addressCity': 'Glasgow',
+          'addressRegion': 'Scotland',
+          'postalCode': 'G1 2FFF',
+          'addressCountry': 'CA'
+        },
+      'mailingAddress':
+        {
+          'streetAddress': '220 Buchanan St',
+          'addressCity': 'Glasgow',
+          'addressRegion': 'Scotland',
+          'postalCode': 'G1 2FFF',
+          'addressCountry': 'CA'
+        }
     }
-    store.state.deliveryAddress = null
+    store.state.recordsAddress = null
 
-    Vue.nextTick(() => {
-      expect(vm.mailingAddress).not.toBeNull()
-      expect(vm.deliveryAddress).toBeNull()
-      expect(vm.$el.querySelectorAll('.v-list-item').length).toEqual(1)
-      expect(vm.$el.querySelector('.v-list-item__title').textContent).toBe('Mailing Address')
-      expect(vm.$el.querySelector('.address-info').textContent).toContain('Victoria BC')
+    const wrapper = mount(AddressListSm, { store, vuetify })
+    const vm = wrapper.vm as any
 
+    Vue.nextTick(async () => {
+      expect(vm.registeredAddress).toBeDefined()
+      expect(vm.recordsAddress).toBeNull()
+      expect(vm.$el.querySelector(
+        '.address-panel .address-info').textContent).toContain('Glasgow')
+      expect(vm.$el.querySelector(
+        '.address-panel .address-info li:nth-child(4)').textContent).toContain('Canada')
+
+      wrapper.destroy()
       done()
     })
   })
 
-  it('displays the delivery address', done => {
-    // init store
-    store.state.mailingAddress = null
-    store.state.deliveryAddress = {
-      'streetAddress': '220 Buchanan St',
-      'addressCity': 'Glasgow',
-      'addressRegion': 'Scotland',
-      'postalCode': 'G1 2FFF',
-      'addressCountry': 'UK'
+  it('displays registered address and records address data when a Bcorp', async done => {
+    // Init Store
+    store.state.entityType = EntityTypes.BCorp
+    store.state.registeredAddress = {
+      'deliveryAddress':
+        {
+          'streetAddress': '220 Buchanan St',
+          'addressCity': 'Glasgow',
+          'addressRegion': 'Scotland',
+          'postalCode': 'G1 2FFF',
+          'addressCountry': 'CA'
+        },
+      'mailingAddress':
+        {
+          'streetAddress': '220 Buchanan St',
+          'addressCity': 'Glasgow',
+          'addressRegion': 'Scotland',
+          'postalCode': 'G1 2FFF',
+          'addressCountry': 'CA'
+        }
     }
+    store.state.recordsAddress = {
+      'deliveryAddress':
+        {
+          'streetAddress': '123 Cloverdale St',
+          'addressCity': 'Victoria',
+          'addressRegion': 'BC',
+          'postalCode': 'G1 2FFF',
+          'addressCountry': 'CA'
+        },
+      'mailingAddress':
+        {
+          'streetAddress': '321 Burrard St',
+          'addressCity': 'Vancouver',
+          'addressRegion': 'BC',
+          'postalCode': 'G1 2FFF',
+          'addressCountry': 'CA'
+        }
+    }
```

Comment:
Can you also add a test case for records address "Same as above"?

Reply:
Done!

Reply:
Done, thanks.